docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -21,6 +21,7 @@
 | DCLW (*)      |        256 |    200 |              0.755 |   53.7 Min |      4.3 GByte |
 | DINO (Res18)  |        256 |    200 |              0.736 |   86.5 Min |      4.1 GByte |
 | MSN (ViT-S)   |        256 |    200 |              0.741 |   92.7 Min |     16.3 GByte |
+| SMoG          |        128 |    200 |              0.698 |  220.9 Min |     14.3 GByte |
```

Comment:
Could you add the result between SimSiam and SwaV to keep alphabetical ordering?

Reply:
Also, is it not possible to train with batch size 256?

Reply:
Probably a batch size of 256 would need too much memory.